app.json

```diff
@@ -24,12 +24,15 @@
   ],
   "environments": {
     "review": {
-    "addons": [
-      "newrelic",
-      "papertrail",
-      "memcachier",
-      "heroku-postgresql"
-      ]
+      "addons": [
+        "newrelic",
+        "papertrail",
+        "memcachier",
+        "heroku-postgresql"
+      ],
+      "scripts": {
+        "postdeploy": "rake db:migrate"
```

Comment:
So if we ever add an environment we should expect that to fail too, right? I guess we can cross that bridge when we get to it

Reply:
yeah I think it might but it might also have to do with review apps and their dbs specifically, I don't know. Kinda was planning to just cross it when we get to it